src/quote/wrapped/index.test.ts

```diff
@@ -15,7 +13,11 @@ const indexToken: QuoteToken = {
 }
 
 describe('WrappedQuoteProvider()', () => {
-  test('it works', async () => {
+  beforeEach((): void => {
+    jest.setTimeout(100000000)
+  })
+
+  test('returns a quote for minting MMI', async () => {
```

Comment:
This test currently fails with: `call revert exception; VM Exception while processing transaction: reverted with reason string "Unexpected error"`